modules/log6x-6-1-0-rn.adoc

```diff
@@ -0,0 +1,53 @@
+= Logging 6.1.0 Release Notes
```

Comment:
@libander I'm merge reviewing the Differntiation [PR](https://github.com/openshift/openshift-docs/pull/84861/files#diff-2620632a1c41ceaaaa2cc8438f93892424205fcd52cdab1d11cf904bbf08bff0), and I am wondering if the 6.1 RN file there, which is titled 6.0.0, and this file(https://github.com/openshift/openshift-docs/pull/84861/files#diff-2620632a1c41ceaaaa2cc8438f93892424205fcd52cdab1d11cf904bbf08bff0) will clash?

Reply:
cc @smunje1 @briandooley